docs/build-acceleration.md

```diff
@@ -59,8 +72,10 @@ For example, if `MyPackage` is known to be incompatible with Build Acceleration,
 <Project>
   <PropertyGroup>
     <AccelerateBuildsInVisualStudio>true</AccelerateBuildsInVisualStudio>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
   </PropertyGroup>
   <ItemGroup>
+    <!-- Disable Build Acceleration for projects that reference specific incompatible packages. -->
```

Comment:
(General question; don't hold the PR for this.) If the solution uses Central Package Management would it make sense to put the `BuildAccelerationIncompatiblePackage` in the Directory.Packages.props file instead?

Reply:
That's an interesting idea. I guess it'd work too. Depends whether you want the list near the packages, or near the properties that control Build Acceleration. I'll say that I don't expect use of these items to be very common. They're detailed here for completeness.